### ISSUE TITLE
[React Native]  Warning Unchecked cast: Any? to Map<String, Any?>?

### ISSUE THREAD

**Issue by Andarius** [bugsnag/bugsnag-js#1004]
When running `npx react-native run-android`, the following warning message appears:

```
> Task :bugsnag_react-native:compileDebugKotlin
w: my-app/node_modules/@bugsnag/react-native/android/src/main/java/com/bugsnag/android/BugsnagReactNative.kt: (75, 84): Unchecked cast: Any? to Map<String, Any?>?
w:  my-app/node_modules/@bugsnag/react-native/android/src/main/java/com/bugsnag/android/BugsnagReactNative.kt: (76, 88): Unchecked cast: Any? to Map<String, Any?>?
```

and 

```
w:  my-app/node_modules/@bugsnag/react-native/android/src/main/java/com/bugsnag/android/MapExtensions.kt: (19, 79): Unchecked cast: Any? to MutableMap<String, Any>
```

**Packages**
```
 "@bugsnag/react-native": "^7.3.2",
 "react": "16.13.1",
 "react-native": "0.63.0"
```

**Comment by steve-nester-uk** [comment]
Thanks for raising @Andarius, we'll take a look.

**Comment by steve-nester-uk** [comment]
A PR has been raised: https://github.com/bugsnag/bugsnag-js/pull/1027.

**Comment by johnkiely1** [comment]
Hi @Andarius.

This has now been fixed and released in version [ v7.3.4](https://github.com/bugsnag/bugsnag-js/releases/tag/v7.3.4).
